src/components/AddCommentCard.jsx

```diff
@@ -1,4 +1,40 @@
-export default function AddCommentCard() {
+import { useState, useEffect } from "react";
+import { postCommentByArticleId, getUsers } from "../api/api";
+import CommentsByArticle from "./CommentsByArticle";
+
+export default function AddCommentCard({ article_id, comments, setComments }) {
+  const [userCommentInput, setUserCommentInput] = useState({
+    username: "",
+    body: "",
+  });
+  const [users, setUsers] = useState([]);
```

Comment:
Consider having users as userContext so you can access this state throughout your app. At the moment you'll only be able to access it in here and any child components (of which there aren't any), so you'd end up having to do the API call you've got on line 33 in every component where you need it.